src/Apps/Auction/AuctionApp.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Separator, Serif } from "@artsy/palette"
+import { AuctionApp_sale } from "__generated__/AuctionApp_sale.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { trackPageViewWrapper } from "Apps/Order/Utils/trackPageViewWrapper"
+import { ErrorPage } from "Components/ErrorPage"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import {
+  Elements,
+  ReactStripeElements,
+  StripeProvider,
+} from "react-stripe-elements"
+import RegistrationForm from "./Components/RegistrationForm"
+
+declare global {
+  interface Window {
+    Stripe?: (key: string) => stripe.Stripe
+    sd: {
+      STRIPE_PUBLISHABLE_KEY: string
+    }
+  }
+}
+
+interface AuctionAppProps extends ReactStripeElements.InjectedStripeProps {
+  sale: AuctionApp_sale
+  relay?: RelayProp
+}
+
+interface AuctionAppState {
+  stripe: stripe.Stripe
+}
+
+export class AuctionApp extends Component<AuctionAppProps, AuctionAppState> {
+  state: AuctionAppState = { stripe: null }
+  CreditCardInput
+  componentDidMount() {
+    if (window.Stripe) {
+      this.setState({
+        stripe: window.Stripe(window.sd.STRIPE_PUBLISHABLE_KEY),
+      })
+    } else {
+      document.querySelector("#stripe-js").addEventListener("load", () => {
+        // Create Stripe instance once Stripe.js loads
+        this.setState({
+          stripe: window.Stripe(window.sd.STRIPE_PUBLISHABLE_KEY),
```

Comment:
This is how we typically use sharify, as opposed to plucking directly from `window`:  
```js
import { data as sd } from 'sharify'
...
sd. STRIPE_PUBLISHABLE_KEY
```